### ISSUE TITLE
record in use when launching from LMS

### ISSUE THREAD

**Issue by carlycurran** [openstax/tutor#247]
**Describe the bug**
When a student launches from LMS, s/he cannot choose to log in to Tutor. That means, if the LMS-associated email has already been used in a Tutor account, the student gets an error. 

Issue was reported in Blackboard

**Expected behavior**
Student has option to log in, not just create an account

**Screenshots**

**Desktop (please complete the following information):**

### Checklist for Done
2-CODE
  - [ ] A pull request is opened, the automated pull request checks pass, and the change has been approved by other developers.
  - [ ] If there are new acceptance tests, they are finalized in "Given / When / Then" format.
  - [ ] Regression test categories are identified on the issue. Categories should be referenced if any changes might affect them, even if the intended functionality is unchanged.
  
4A-UX REVIEW
  - [ ] The pull request is merged into master, and the change branch is deleted.

5A-FUNCT VER
  - [ ] All the acceptance tests for this issue have passed.
 
5C-REGRESSION
  - [ ] The test plan in testrail passes
  - [ ] Blessing from Product Manager that the release is good to go.

5D-READY TO BE DEPLOYED
  - [ ] Production is updated and verified.
  - [ ] All issues in the release are closed.

**Comment by carlycurran** [comment]
Let's wait on this investigation to get more data -- we need to see if this happens to any other students. Nathan could not replicate in Canvas

**Comment by Malar-Natarajan** [comment]
COuld not replicate the issue in QA.
But the fix to login to existing account works good in staging - blackboard

**Comment by Malar-Natarajan** [comment]
Verified in Staging canvas
